apis/workstations/v1alpha1/workstationcluster_types.go

```diff
@@ -139,7 +139,7 @@ type WorkstationClusterObservedState struct {
 	//  workstation cluster. Workstation VMs need access to this IP address to work
 	//  with the service, so make sure that your firewall rules allow egress from
 	//  the workstation VMs to this address.
-	ControlPlaneIp *string `json:"controlPlaneIp,omitempty"`
+	ControlPlaneIP *string `json:"controlPlaneIP,omitempty"`
```

Comment:
Nice! I think then we can remove the exception from the apichecks?

Reply:
There was no exception for this I think -- had to make the change because it was failing the test.